java/vector/src/main/java/org/apache/arrow/vector/BaseFixedWidthVector.java

```diff
@@ -48,7 +48,8 @@
         implements FixedWidthVector, FieldVector, VectorDefinitionSetter {
   private final int typeWidth;
 
-  protected int lastValueCapacity;
+  protected int lastTargetValueCapacity;
```

Comment:
I am not sure if we should change this variable's name, as in other classes (e.g. BaseVariableWidthVector and BaseLargeVariableWidthVector), they are called "lastValueCapacity"

Reply:
Sure, I'll change it back! I was a bit worried because of the proximity to actualValueCapacity, but I didn't realized it was used elsewhere and had a larger meaning.

Reply:
Changed